test/unit/model/qti/ParserFactoryTest.php

```
@@ -146,4 +246,152 @@ private function readSampleFile(string $name): string
+XML;
+        return "<root>" . $xml . "</root>";
+    }
+
+    private function getParseItemWithIsNullParser(string $xml)
+    {
+        $itemDocument = $this->readSampleFile('testParseItem_itemDocument.xml');
+        $dom = new DOMDocument();
+        $dom->loadXML($itemDocument);
+
+        $parser = new class ($dom) extends ParserFactory {
+            public $test_isCalled_buildIsNullFeedbackRule = false;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $test_isCalled_buildIsNullFeedbackRule. Keep variable name length under 20. (Naming Rules)

(LongVariable)